google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1133,6 +1144,11 @@ public Builder enableGrpcGcpExtension() {
     public Builder enableGrpcGcpExtension(GcpManagedChannelOptions options) {
       this.grpcGcpExtensionEnabled = true;
       this.grpcGcpOptions = options;
+      // By default, set number of channels to GRPC_GCP_ENABLED_DEFAULT_CHANNELS if gRPC-GCP
+      // extension is enabled
```

Comment:
Is there a better comment possible here? Think along lines of, when can `gRPC-GCP` extension be enabled.

Reply:
Does this comment look good?
By default, set number of channels to GRPC_GCP_ENABLED_DEFAULT_CHANNELS, when gRPC-GCP extension is enabled to improve resiliency during network issues and to provide additional channel metrics